src/test/java/org/eclipse/cargotracker/domain/model/cargo/CargoTest.java

```diff
@@ -16,8 +13,10 @@
 import org.eclipse.cargotracker.domain.model.location.SampleLocations;
 import org.eclipse.cargotracker.domain.model.voyage.Voyage;
 import org.eclipse.cargotracker.domain.model.voyage.VoyageNumber;
-import org.junit.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Let's not do wildcard imports as per Google Style Guide: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports